models/classes/class.QtiTestCompiler.php

```diff
@@ -793,14 +792,8 @@ protected function copyPublicResources()
 
             if (in_array($mime, self::getPublicMimeTypes()) === true && $pathinfo['extension'] !== 'php') {
                 $publicPathFile = $testDefinitionDir->getRelPath($file);
-                try {
-                    common_Logger::d('Public ' . $file->getPrefix() . '(' . $mime . ') to ' . $publicPathFile);
-                    $publicCompiledDocDir->getFile($publicPathFile)->write($file->readStream());
-                } catch (FileExistsException $e) {
-                    common_Logger::w(
-                        'File ' . $publicPathFile . ' copied twice to public test folder during compilation'
-                    );
-                }
+                common_Logger::d('Public ' . $file->getPrefix() . '(' . $mime . ') to ' . $publicPathFile);
+                $publicCompiledDocDir->getFile($publicPathFile)->write($file->readStream());
```

Comment:
Thanks for checking, that exception does not exist anymore in Flysystem library, and write method will overwrite existing object if it happens to be there. I thought that it would be better to just overwrite, rather than making additional call to  check if file already exists